avahi-core/iface-linux.c

```diff
@@ -194,6 +195,8 @@ static void netlink_callback(AvahiNetlink *nl, struct nlmsghdr *n, void* userdat
         if (!(i = avahi_interface_monitor_get_interface(m, (AvahiIfIndex) ifaddrmsg->ifa_index, avahi_af_to_proto(ifaddrmsg->ifa_family))))
             return;
 
+        memset(label, 0, IFNAMSIZ + 1);
```

Comment:
Yes, it is not a big deal. <pre>char label[IFNAMSIZ + 1] = "";</pre> have also the same effect of zeroing out the whole array without using a separate memset.